README.md

```diff
@@ -15,4 +15,39 @@ Plugin-Repository/Repository/{Version}/{Owner}/{Repository}/{Name}.json
\ No newline at end of file
+| Name       | The name of the plugin                                      |
+
+## Plugin Metadata
+
+| Key                                  | Value                                                                                                                                                                                                                  |
+|--------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Name                                 | The plugin's name.                                                                                                                                                                                                     |
+| Owner                                | The username of the plugin owner.                                                                                                                                                                                      |
+| Description                          | A brief description of the plugin's function.                                                                                                                                                                          |
+| PluginVersion                        | The plugin version in four-part version number format.                                                                                                                                                                 |
+| SupportedDriverVersion               | The minimum required OpenTabletDriver version for the plugin to function in four-part version number format.                                                                                                           |
+| (Optional) MaxSupportedDriverVersion | The maximum OpenTabletDriver version the plugin functions on in four-part version number format. This should only be included in cases where a minor OpenTabletDriver version introduced breaking changes to a plugin. |
+| RepositoryUrl                        | The git repository url containing the plugin's source code.                                                                                                                                                            |
+| DownloadUrl                          | The direct download url for the plugin's release files in a compressed archive format. The plugin's dll files must be contained in the top-level directory of the compressed archive.                                  |
+| CompressionFormat                    | The compression format used by the compressed archive in the DownloadUrl field. `zip` is preferred.                                                                                                                    |
```

Comment:
```suggestion
| CompressionFormat                    | The compression format used by the compressed archive in the DownloadUrl field. Only `zip` is supported right now.                                                                                                                    |
```

Reply:
"currently" instead?

Reply:
Only `zip` is currently supported?

Reply:
Yeah seems better imo

Reply:
Changed to "Currently, `zip` is the only supported value."